airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumerTest.java

```diff
@@ -315,7 +315,7 @@ private static void consumeRecords(final BufferedStreamConsumer consumer, final
     List<AirbyteMessage> output = Lists.newArrayList();
     long bytesCounter = 0;
     for (int i = 0;; i++) {
-      JsonNode payload = Jsons.jsonNode(ImmutableMap.of("id", RandomStringUtils.randomAscii(7), "name", "human " + String.format("%5d", i)));
+      JsonNode payload = Jsons.jsonNode(ImmutableMap.of("id", RandomStringUtils.randomAlphabetic(7), "name", "human " + String.format("%8d", i)));
```

Comment:
how does this make the generated size always be 40?

Reply:
now it generates object of fixed size:
```
{"id":"dumplEw","name":"human        0"}
{"id":"tnJatrn","name":"human       25"}
```

I have added more spaces before number (to make it 40 bytes)

I have also changed `randomAscii` -> `randomAlphabetic`
to avoid generation BOULD-QUOTE and BACK-SLACH which can generated 2 bytes after json-encoding

Reply:
Got it, thanks for the explanation!